src/bin/FacilitatorStart.ts

```diff
@@ -0,0 +1,168 @@
+import { Config, FacilitatorConfig } from '../Config';
+import MosaicConfig from '../MosaicConfig';
+import { FacilitatorStartException } from '../Exception';
+
+/**
+ * It parses and evaluates input params of facilitator start command to get config object.
+ */
+export default class FacilitatorStart {
+  /**
+   * It provides Config object encapsulating mosaic and facilitator configs.
+   */
+  public static getConfig(originChain: string, auxChain: string, options: any): Config {
+    let configObj = {} as any as Config;
+    let onlyFacilitator = false;
+    if (
+      originChain === undefined && auxChain === undefined && options.facilitatorConfig !== undefined
+    ) {
+      onlyFacilitator = true;
+      configObj = FacilitatorStart.facilitatorConfigOption(options);
+    }
+
+    if (onlyFacilitator === false) {
+      FacilitatorStart.isOriginAuxChainDefined(originChain, auxChain);
+      configObj = FacilitatorStart.originAuxChainOption(originChain, auxChain, options);
+    }
+
+    return configObj;
+  }
+
+  /**
+   * This method provides Config object if origin chain and aux chain is defined.
+   * @param {string} originChain Chain name of the origin.
+   * @param {string} auxChain Chain id of the auxiliary.
+   * @param options It contains path to facilitator and mosaic config.
+   * @returns {Config} Config object encapsulating facilitator and mosaic configs.
+   */
+  private static originAuxChainOption(originChain: string, auxChain: string, options: any): Config {
+    let configObj = {} as any as Config;
+    if (originChain !== undefined && auxChain !== undefined) {
```

Comment:
There are so many if else conditions here which are nested deep. This is not readable and hard to maintain.
Use short circuit and early return approach. raw example is below:

```
    if(!originChain && !auxChain) {
      return configObj;
    }
    
    if(options.facilitatorConfig && options.mosaicConfig) {

      const facilitator: FacilitatorConfig = FacilitatorConfig.from(auxChain);
      const mosaic: MosaicConfig = MosaicConfig.fromChain(originChain);
      return new Config(mosaic, facilitator);
    }
```
https://stackoverflow.com/questions/9344305/what-is-short-circuiting-and-how-is-it-used-when-programming-in-java

Follow this approach in other methods of this class also.